README.md

```diff
@@ -1 +1,56 @@
+| ----------- | ----------------------------------------------------- | ----------------------------------------------------- |
+| kord        | Idiomatic Discord API wrapper                         | [kord](https://github.com/kordlib/kord)               |
+| cache       | Caching library used by Kord                          | [cache](https://github.com/kordlib/cache)             |
+| codegen.kt  | Library we use for generating code                    | [codegen-kt](https://github.com/kordlib/codegen-kt)   |
+| .teamcity   | CI Configuration                                      | [.teamcity](https://github.com/kordlib/.teamcity)     |
+| kordx.emoji | Kotlin emoji library, designed for usage with Discord | [kordx.emoji](https://github.com/kordlib/kordx.emoji) |
+
+## License
+
+Unless stated otherwise in the projects Repository our work is licensed under the [MIT License](https://choosealicense.com/licenses/mit/)
+
+## Code of Conduct
+This code of conduct outlines our expectations for all those who want to participate or interact with the Kord project via GitHub Issues, Discussions, Pull Requests, our Discord server or simmilar
+### How to behave
+The following behaviours are expected and requested of all community members:
+
+- Participate in an authentic and active way. In doing so, you contribute to the health and longevity of this community.
+- Exercise consideration, respect and empathy in your speech and actions. Remember, we have all been through different stages of learning when adopting technologies.
+- Refrain from demeaning, discriminatory, or harassing behaviour and speech.
+- Disagreements on things are fine, argumentative behaviour or trolling are not.
+
+### How not to behave
+- Do not perform threats of violence or use violent language directed against another person.
+- Do not make jokes of sexist, racist, homophobic, transphobic, ableist or otherwise discriminatory nature, or use language of this nature.
+- Do not post or display sexually explicit or violent material.
+- Do not post or threaten to post other people’s personally identifying information ("doxing").
+- Do not make personal insults, particularly those related to gender, sexual orientation, race, religion, or disability.
+- Do not engage in sexual attention. This includes, sexualised comments or jokes and sexual advances.
+- Do not advocate for, or encourage, any of the above behaviour.
+
+Please take into account that online communities bring together people from many different cultures and backgrounds. It's important to understand that sometimes the combination of cultural differences and online interaction can lead to misunderstandings. That is why having empathy is very important.
+
+### How to report issues
+If you want to report violations of another member and/or need assitance because of that persons behavior feel free to contact one of the Admins (schlaubi, lustiger.lurch) on [our Discord Server](https://discord.gg/6jcx5ev)
```

Comment:
`violations of another member` probably isn't the wording you want here. Makes it sound like we're sticking them with a knife.

Perhaps `another member's violations`, or `violations of this Code of Conduct`?

You're also missing some punctuation -  `that person's behaviour, feel free`

Also, `assitance` isn't a word. You should run LanguageTool over this, which you can do in IntelliJ via the Grazie Pro plugin.